cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -498,6 +498,23 @@ def _unitary_(self) -> np.ndarray:
             return m
         raise ValueError(f'{self} is not unitary')
 
+    def _json_dict_(self):
+        def key_json(k):
+            # k is a frozenset, each element of frozen set is a tuple.
+            return [[x for x in e] for e in sorted(list(k))]
```

Comment:
`sorted(list(...))` is redundant: can just do `sorted()`

`[x for x in e]` can just be `list(e)`

by default, python can serialize tuples. It just makes them lists, so you have to take care when de-serializing. 


Final:
```python
return sorted(k)
```

Reply:
done